topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.md

```diff
@@ -168,9 +168,9 @@ You can access the data for this tutorial in multiple ways:
 > 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.10.3+galaxy0) %} with the following parameters:
 >    - {% icon param-file %} *"Annotated data matrix"*: `N701-400k`
 >    - *"Function to manipulate the object"*: `Concatenate along the observations axis`
->    - {% icon param-file %} *"Annotated data matrix to add"*: `Select all the other matrix files from bottom to top, N707 to N702`
+>    - {% icon param-file %} *"Annotated data matrix to add"*: `Select all the other matrix files from bottom to top, N702 to N707` (if you imported files from Zenodo, your history may not be in this order, but you can still select them in order from N702 to N707 using the file names)
```

Comment:
I don't understand the sentence in the brackets? What means selecting in order? They are all selected together, no?

Reply:
I've tried to make this clearer in the new version - the point is that they're concatenated in the order you click and as we'll use the batch numbers to add sample metadata, if they're not in the same order as the tutorial it will cause problems!